<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAcolitoAsistentes</class>
 <widget class="QWidget" name="MyAcolitoAsistentes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="searchComboBox">
           <item>
            <property name="text">
             <string>Nombre</string>
            </property>
            <property name="icon">
             <iconset resource="acolitoRC.qrc">
              <normaloff>:/acolitos/iconos/buscar</normaloff>:/acolitos/iconos/buscar</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DNI</string>
            </property>
            <property name="icon">
             <iconset resource="acolitoRC.qrc">
              <normaloff>:/acolitos/iconos/buscar</normaloff>:/acolitos/iconos/buscar</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Email</string>
            </property>
            <property name="icon">
             <iconset resource="acolitoRC.qrc">
              <normaloff>:/acolitos/iconos/buscar</normaloff>:/acolitos/iconos/buscar</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchLine"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="vistaAsistentes">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="visualizacionBox">
         <property name="title">
          <string>Mostrar opciones de visualizacion</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="pagoLabel">
            <property name="text">
             <string>Han pagado:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="siPagoCheckBox">
            <property name="text">
             <string>Sí</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="noPagoCheckBox">
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="llegadaLabel">
            <property name="text">
             <string>Han llegado:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="siLlegadaCheckBox">
            <property name="text">
             <string>Sí</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="noLlegadaCheckBox">
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="asistenteWidget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#asistenteWidget
{
	background-color: qlineargradient(spread:pad, x1:0.449, y1:0.443, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 170, 0, 255));
   border: 1px solid grey;
   border-radius: 10px;
}

QLineEdit
{
 
	background-color: rgba(255, 255, 255, 0);
    border: no;
    
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QTextBrowser
{
 
	background-color: rgba(255, 255, 255, 0);
    border: no;
    
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QLineEdit#nombreLine
{
	color: rgb(255, 170, 0);
}

QLabel
{
	background-color: rgba(255, 255, 255, 0);
    border: no;
    
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QComboBox
{
	background-color: rgba(255, 255, 255, 0);
    border: no;
    
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QComboBox::drop-down
{
 border-right: no
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="AcolitoBinaryToolButton" name="botonActivo">
           <property name="minimumSize">
            <size>
             <width>62</width>
             <height>62</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>62</width>
             <height>62</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Pulsa para indicar que la persona ha llegado y ha recogido su acreditación</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="acolitoRC.qrc">
             <normaloff>:/acolitos/iconos/falseIcon</normaloff>:/acolitos/iconos/falseIcon</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>52</width>
             <height>52</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="AcolitoVariantButton" name="imprimirButton">
             <property name="toolTip">
              <string>Imprimir asistente</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/imprimir</normaloff>:/acolitos/iconos/imprimir</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="habButton">
             <property name="toolTip">
              <string>Alojar o cambiar al grupo de habitacion</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/habitacion</normaloff>:/acolitos/iconos/habitacion</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="AcolitoVariantButton" name="grupoButton">
             <property name="toolTip">
              <string>Cambiar al asistente de habitacion.</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/editarGrupo</normaloff>:/acolitos/iconos/editarGrupo</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="separarButton">
             <property name="toolTip">
              <string>Separa al asistente de su grupo</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/separar</normaloff>:/acolitos/iconos/separar</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="AcolitoVariantButton" name="exportarButton">
             <property name="toolTip">
              <string>Guarda los datos del asistente en PDF o en ODT (Libre Office y MS Word)</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/exportar</normaloff>:/acolitos/iconos/exportar</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="idLabel">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
    border: 1px solid grey;
    border-radius: 5px;
    
	background-color: rgb(255, 170, 0);
	color: rgb(255, 255, 255);
}
</string>
             </property>
             <property name="text">
              <string>()</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nombreLine">
             <property name="text">
              <string>Nombre</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>DNI:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="dniLine">
             <property name="text">
              <string>DNI</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Teléfono:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="telefonoLine">
             <property name="text">
              <string>Teléfono</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Correo:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="mailLine">
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Categoría:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="AcolitoTextComboBox" name="categoriasComboBox"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Comentarios:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" rowspan="2">
            <widget class="QTextBrowser" name="comentariosBrowser">
             <property name="font">
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;comentarios&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Habitación:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="habLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	color: rgb(0, 170, 0);
}</string>
             </property>
             <property name="text">
              <string>codHab</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="llaveButton">
             <property name="text">
              <string>Entregar llave</string>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/habAbierta</normaloff>:/acolitos/iconos/habAbierta</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Compañeros de habitación:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="vistaGrupo">
             <property name="styleSheet">
              <string notr="true">QLineEdit
{
	font: 9pt &quot;Segoe UI&quot;;
}

QComboBox
{
	font: 9pt &quot;Segoe UI&quot;;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AcolitoBinaryToolButton</class>
   <extends>QToolButton</extends>
   <header>acolitobinarytoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AcolitoVariantButton</class>
   <extends>QToolButton</extends>
   <header>acolitovariantbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AcolitoTextComboBox</class>
   <extends>QComboBox</extends>
   <header>acolitotextcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="acolitoRC.qrc"/>
 </resources>
 <connections/>
</ui>
